File: Projekt_Python/rosliny/Jagoda.py
from Roslina import Roslina

class Jagoda (Roslina): 
    def __init__(self, x, y,dane):
        if dane==None:
            self._x = x
            self._y = y
            self._symbol = 'j'
            self._sila = 99
            self._wykonal_ruch = False
            self._inicjatywa = 0
            self._zycie = 0
            self._numer=3
        else:
            self._x = dane[0]
            self._y = dane[1]
            self._inicjatywa=dane[2]
            self._symbol = 'j'
            self._sila = dane[3]
            self._wykonal_ruch = False
            self._zycie = dane[4]
            self._numer=dane[5]
        self._kolor= (0,0,255)

File: projekt_java/src/po/java/rosliny/Jagoda.java
package po.java.rosliny;

import po.java.Roslina;

import java.awt.*;

public class Jagoda extends Roslina {
    public Jagoda(int x, int y) {
        this.x = x;
        this.y = y;
        this.symbol = 'j';
        this.sila = 0;
        this.wykonal_ruch = false;
        this.zycie = 0;
        this.numer=3;
        this.kolor= Color.BLUE;
    }
    public Jagoda(int []dane)
    {
        this.x = dane[0];
        this.y = dane[1];
        this.inicjatywa=dane[2];
        this.symbol = 'j';
        this.sila = dane[3];
        this.wykonal_ruch = false;
        this.zycie = dane[4];
        this.numer=dane[5];
        this.kolor= Color.BLUE;
    }
}
